county.tmpl

```diff
@@ -44,4 +44,15 @@ module County {
     | Baron
     | Etc
   ]
+
+  // A date, encoded per RFC 3339.
+  type Date: wrap string
+    <scala
+      t="org.joda.time.LocalDate"
+      wrap="gov.wicourts.common.LocalDateOps.TODO:R-16899"
+      unwrap="gov.wicourts.common.LocalDateOps.sortableDateFormat.print">
+    <purs
+      t="Ccap.Common.DateTime.Date"
+      wrap="Ccap.Common.DateTime.Date.fromIsoMadison"
+      unwrap="Ccap.Common.DateTime.Date.inIso8601ExtendedDateFormat">
```

Comment:
This example is not meant to preclude later allowing a `codec=` annotation for when we do have consistent naming.

Reply:
Actually, enforcing that for PS at least might let us avoid/defer naming issues...